[PATCH] Remove /usr/local paths

remove /usr/local paths from all stuff thats used
in Configure script

Patch-Name: gentoo/usr_local.diff
Original-By: Torsten Veller <tove@gentoo.org>

diff -ruN perl-5.24.0-RC5.orig/Configure perl-5.24.0-RC5/Configure
--- perl-5.24.0-RC5.orig/Configure	2016-04-27 00:11:09.000000000 +0200
+++ perl-5.24.0-RC5/Configure	2016-05-05 23:36:35.518843279 +0200
@@ -1425,8 +1425,7 @@
 i_whoami=''
 : Possible local include directories to search.
 : Set locincpth to "" in a hint file to defeat local include searches.
-locincpth="/usr/local/include /opt/local/include /usr/gnu/include"
-locincpth="$locincpth /opt/gnu/include /usr/GNU/include /opt/GNU/include"
+locincpth=""
 :
 : no include file wanted by default
 inclwanted=''
@@ -1442,12 +1441,11 @@
 : change the next line if compiling for Xenix/286 on Xenix/386
 xlibpth='/usr/lib/386 /lib/386'
 : Possible local library directories to search.
-loclibpth="/usr/local/lib /opt/local/lib /usr/gnu/lib"
-loclibpth="$loclibpth /opt/gnu/lib /usr/GNU/lib /opt/GNU/lib"
+loclibpth=""
 
 : general looking path for locating libraries
 glibpth="/lib /usr/lib $xlibpth"
-glibpth="$glibpth /usr/ccs/lib /usr/ucblib /usr/local/lib"
+glibpth="$glibpth /usr/ccs/lib /usr/ucblib"
 test -f /usr/shlib/libc.so && glibpth="/usr/shlib $glibpth"
 test -f /shlib/libc.so     && glibpth="/shlib $glibpth"
 test -d /usr/lib64         && glibpth="$glibpth /lib64 /usr/lib64 /usr/local/lib64"
@@ -5031,7 +5029,7 @@
 : Set private lib path
 case "$plibpth" in
 '') if ./mips; then
-	plibpth="$incpath/usr/lib $sysroot/usr/local/lib $sysroot/usr/ccs/lib"
+	plibpth="$incpath/usr/lib $sysroot/usr/ccs/lib"
     fi;;
 esac
 case "$libpth" in
@@ -21798,7 +21796,7 @@
 	case "$ranlib" in
 	:) ranlib='';;
 	'')
-		ranlib=`./loc ranlib X /usr/bin /bin /usr/local/bin`
+		ranlib=`./loc ranlib X /usr/bin /bin`
 		$test -f $ranlib || ranlib=''
 		;;
 	esac
@@ -24964,8 +24962,8 @@
 ln='$ln'
 lns='$lns'
 localtime_r_proto='$localtime_r_proto'
-locincpth='$locincpth'
-loclibpth='$loclibpth'
+locincpth=''
+loclibpth=''
 longdblinfbytes='$longdblinfbytes'
 longdblkind='$longdblkind'
 longdblmantbits='$longdblmantbits'
